docs/index.rst

```diff
@@ -0,0 +1,145 @@
+Overview
+########
+
+Getting started
+===============
+
+Buildhub is a Kinto collection that contains the information about all
+the builds.
+
+**Browsing using Kinto**
+
+You can access the database using the Kinto API:
+
+.. note::
+   
+    https://kinto-ota.dev.mozaws.net/v1/buckets/build-hub/collections/releases/records?_limit=10
+
+`More information in the Kinto documentation <http://kinto.readthedocs.io/en/stable/api/1.x/index.html#full-reference>`_.
+
+**Browsing using Elasticsearch**
+
+You can also use an ElasticSearch endpoint for more powerfull queries:
```

Comment:
powerful

Most -ful suffixes are with one l. There are some exceptions -- overfull and underfull, using full as an adjective; bellyfull or bucketfull, using -full as a measure.

```
> grep '.*ful$' /usr/share/dict/words | wc
   1086    1086   10744
> grep '.*full$' /usr/share/dict/words | wc
     34      34     332
```